packages/graphql/src/schema/question.ts

```diff
@@ -79,6 +79,9 @@ export const OptionsFreeTextInput = builder.inputType('OptionsFreeTextInput', {
 })
 
 export const ResponseInput = builder.inputType('ResponseInput', {
+  directives: {
```

Comment:
this together with the directives plugin will allow to ensure that exactly one type of response is sent (not empty and not free and choice response together)